scripts/load_kube_credentials.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Function to check if a command exists
+not_installed() {
+    ! type "$1" &> /dev/null
+}
+
+# Check for jq
+if not_installed jq; then
+    echo "jq is not installed. You can install it from: https://stedolan.github.io/jq/download/"
+    exit 1
+fi
+
+# Check for yq
+if not_installed yq; then
+    echo "yq is not installed. You can install it from: https://github.com/mikefarah/yq"
+    exit 1
+fi
+
+# Check for op
+if not_installed op; then
+    echo "op is not installed. You can install it from: https://developer.1password.com/docs/cli/get-started/"
+    exit 1
+fi
+
+# The string to match with context.user
+USERNAME="$1"
```

Comment:
MAYBE: Same comment as below, we could use only the context name
https://github.com/OpenMined/PySyft/pull/8199/files#r1461342083

Reply:
Ohh, I got what you meant! Thanks!
Solved by: [c7d25f1](https://github.com/OpenMined/PySyft/pull/8199/commits/c7d25f1d5cffd4c76f7666a27f1d27ada504d56f)